### ISSUE TITLE
Make navbar similar to napari.org

### ISSUE THREAD

**Issue by melissawm** [napari/island-dispatch#4]
Also include link to blog in the main page.

**Comment by melissawm** [comment]
One thing to explore is to have an issue opened every time a change to the navbar happens in napari/docs.

**Comment by psobolewskiPhD** [comment]
I assume the blog would go in the top nav bar.
But will it be too busy? Should we maybe consolidate something or use a dropdown? 
Pydata-sphinx page has a `More 🔽 `

**Comment by melissawm** [comment]
Yes @psobolewskiPhD that's what I suggested to @jni yesterday - I think we could even go as far as add other napari-related sites under the "More" rubric, like napari-animation, npe, napari-plugin-manager, napari-workshop-template etc
